src/main/java/com/thoughtworks/pacman/core/Game.java

```diff
@@ -13,6 +13,7 @@
     private final Pacman pacman;
     private final Ghosts ghosts;
     private final PacmanTileVisitor pacmanTileVisitor;
+    private int levelA = 1;
```

Comment:
Neden levelA?

A'nın anlamı ne?

Reply:
"level" ' in daha önce kullanılmış olabilecek çok genel bir değişken ismi olduğunu düşündüğüm için özelleştirmek adına böyle bir yola başvurdum.